src/components/ui/spinners/Spinners.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div class="row">
+    <vuestic-widget :headerText="$t('spinners.title')" class="col-sm-12">
+        <div v-for="(group, i) in groups" :key="i" class="row">
+          <div v-for="item in group" :key="item" class="col">
+            <component
+              :is="item"
+              :size="config.size"
+              :color="config.color"
+              :animation-duration="config.animation"
+              class="spinner-item">
+            </component>
+          </div>
+        </div>
+        <div class="row align-center">
+          <div class="col text-center">
+            Powered by
+            <a :href="'http://epic-spinners.epicmax.co/'" target="_blank">Epic Spinners</a>
+          </div>
+        </div>
+    </vuestic-widget>
+  </div>
+</template>
+
+<script>
+  import * as spinners from 'epic-spinners'
+
+  export default {
+    components: {
+      ...spinners
+    },
+    data: function () {
+      return {
+        config: {
+          animation: 1500,
+          color: '#1ec260',
+          size: 50,
```

Comment:
play with the sizes and margins/paddings. Not sure but maybe they'll look better if they are larger?